### PR TITLE
add mpd and local to the dockerfile

### LINKED ISSUE #31: mpd and local are no longer core to mopidy
I think these need to be added as extensions now

https://docs.mopidy.com/en/latest/changelog/#mpd-frontend

https://github.com/mopidy/mopidy-mpd

### PR DESCRIPTION
**Pull request by marklester**
I added mpd and local via pip to the docker file
closes #31

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -40,7 +40,8 @@ RUN set -ex \
 RUN set -ex \
  && mkdir -p /var/lib/mopidy/.config \
  && ln -s /config /var/lib/mopidy/.config/mopidy
-
+RUN python3 -m pip install Mopidy-Local
```
Comment (christianlent) [inline Dockerfile:43]: I wonder if you could use Pipfile instead (https://github.com/wernight/docker-mopidy/blob/master/Pipfile) so pipenv would pick it up?

**Comment by martinezpenya** [comment]
This will solve also my issue? #34 
After installin mopidy-mpd I can connect to mpd server on 6600, but after install mopidy-local I can't browse local files

**Comment by ahmadnbl** [comment]
> This will solve also my issue? #34
> ~After installing mopidy-mpd I can connect to mpd server on 6600, but after install mopidy-local I can't browse local files~
> Sorry... it's working well... the mpd and also local files. So I think it will solve my issue, can anyone confirm?
> Thank You.

As I commented on https://github.com/wernight/docker-mopidy/issues/34#issuecomment-737212123
it's working with `Home Assistant` after I install the `mpd`

**Comment by sasgoose** [comment]
Is there anything I can do to help get this pull request committed?

**Comment by wernight** [comment]
Merged in the Pipefile PR

**Pull request closed without merging**